v22.1/cost-based-optimizer.md

```diff
@@ -158,6 +158,22 @@ We strongly recommend not setting this value higher than 8 to avoid performance
 
 For more information about the difficulty of selecting an optimal join ordering, see our blog post [An Introduction to Join Ordering](https://www.cockroachlabs.com/blog/join-ordering-pt1/).
 
+### Reduce planning time for plans with many joins
+
+The cost-based optimizer explores multiple join orderings to find the lowest-cost plan. If there are many joins or join subtrees in the query, this can increase the number of execution plans the optimizer explores, and therefore the exploration and planning time. If the plan phase of a query takes a long time (on the order of multiple seconds or minutes) to plan, or the query plan involves many joins, consider the following alternatives to reduce the planning time:
+
+- To limit the size of the subtree that can be reordered, set the `reorder_joins_limit` session variable to a lower value, for example:
```

Comment:
We should make the downsides clear. By reducing `reorder_joins_limit` we reduce the number of plans explored, so a less efficient plan may be chosen by the optimizer.

Reply:
Did you see this comment? I think it's important to highlight the downside of decreasing `reorder_joins_limit`, but let me know if you disagree.

Reply:
I did see the comment and had added the sentence, but it looks like I forgot to commit it. I just did.